generated/nidcpower/session.py

```diff
@@ -32,7 +32,7 @@ class _SessionBase(object):
     # This is needed during __init__. Without it, __setattr__ raises an exception
     _is_frozen = False
 
-    active_advanced_sequence = attributes.AttributeViString(1150074)
+    active_advanced_sequence = attributes.AttributeViString(1150074, converter_to_driver=None, converter_from_driver=None)
```

Comment:
If we had specific types for conversions, then all these ugly changes wouldn't be needed.

Reply:
Switched to additional attribute classes